vhdbuilder/packer/components.json

```diff
@@ -75,8 +75,7 @@
         "1.6.6"
       ],
       "multiArchVersions": [
-        "1.8.6",
-        "v1.8.6"
```

Comment:
don't remove v1.8.6 until we fully rollout v1.8.7

Reply:
done